marlowe-runtime-web/src/Language/Marlowe/Runtime/Web/Contract/State.hs

```diff
@@ -0,0 +1,44 @@
+module Language.Marlowe.Runtime.Web.Contract.State
+  ( ContractState(..)
+  , Payout(..)
+  ) where
+
+
+import Data.Aeson (FromJSON, ToJSON)
+import Data.Map (Map)
+import Data.OpenApi (ToSchema)
+import Data.Text (Text)
+import Data.Word (Word64)
+import GHC.Generics (Generic)
+import qualified Language.Marlowe.Core.V1.Semantics.Types as Semantics
+import Language.Marlowe.Runtime.Web.Orphans ()
+
+import Language.Marlowe.Runtime.Web.Types
+  (BlockHeader, MarloweVersion, Metadata, PolicyId, TextEnvelope, TxOutRef, TxStatus)
+
+data ContractState = ContractState
```

Comment:
Can you please move this back to `Types.hs`? I don't want to have this many types modules. I think we could split it out by `Contracts` `Transactions` and `Withdrawals` for example, but this is too granular. And if we do that, I'd want to refactor everything so it's all consistent.